x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/add_prepackaged_rules_schema.ts

```diff
@@ -82,4 +84,7 @@ export const addPrepackagedRulesSchema = Joi.object({
   references: references.default([]),
   note: note.allow(''),
   version: version.required(),
+
+  // TODO: Remove the hasListsFeatures once this is ready for release
+  lists: hasListsFeature() ? lists.default([]) : lists.forbidden().default([]),
```

Comment:
Curious, what's the `.default([])` do after forbidden? I assumed that by adding forbidden, only `undefined` was allowed?

Reply:
It says that it is forbidden but the value will be an empty array for 7.7 on input. This keeps downstream logic and types simple to be just an empty array for 7.7 with the feature flag even off. That makes it where I don't have to use the feature flag very often and I don't have to worry about it accepting an `undefined/null` since it is an array.

Arrays are typically empty arrays and defaulted when they are empty compared to a value that is a non-array which would be `undefined/null`